### PR TITLE
12.0 backport translation updates

### LINKED ISSUE #353: 14to12: Improve translations
- remove fr_be translation (it is the same as the fr translation, plus translations works additively, we can put only BE specific changes in fr_be, we don't need to maintain a parallel .po file)
- translate mail_templates

Done in v14 (for french), could be done in v12
task : https://gestion.coopiteasy.be/web#id=8962&action=475&active_id=260&model=project.task&view_type=form&menu_id=536

### PR DESCRIPTION
**Pull request by victor-champonnois**
fix https://github.com/coopiteasy/vertical-cooperative/issues/353
tested locally and it seems to work.

### CODE REVIEW

**Comment by victor-champonnois** [comment]
@huguesdk pour info :)

File: cooperator/i18n/cooperator.pot [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # Translation of Odoo Server.
 # This file contains the translation of the following modules:
-#	* cooperator
+# 	* cooperator
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 12.0\n"
+"Project-Id-Version: Odoo Server 14.0\n"
```
Comment (robinkeunen) [inline cooperator/i18n/cooperator.pot:7]: Still 12.0 ?

File: cooperator/i18n/cooperator.pot [hunk 2]
```diff
@@ -15,14 +15,16 @@ msgstr ""
 
 #. module: cooperator
 #: model:mail.template,report_name:cooperator.email_template_release_capital
-msgid "\n"
-"                ${(object.number or '').replace('/','_')}_${object.state == 'draft' and 'draft' or ''}\n"
+msgid ""
```
Comment (robinkeunen) [inline cooperator/i18n/cooperator.pot:18]: The invoice reference field is still `number` in 12.0

File: cooperator/i18n/cooperator.pot [hunk 3]
```diff
@@ -69,194 +71,199 @@ msgstr ""
...
 "\n"
 "    <p>Hello ${object.firstname},</p>\n"
 "\n"
 "    <p>We confirm the reception of you payment for the new share(s) you have taken.</p>\n"
 "\n"
 "    <br/>\n"
-"    <p>Find in attachment your ${object.company_id.name} certificate.</p>\n"
-"    <p>Thank you for trusting ${object.company_id.name or 'us'}!</p>\n"
+"    <p>Find in attachment your ${object.env.company.name} certificate.</p>\n"
+"    <p>Thank you for trusting ${object.env.company.name or 'us'}!</p>\n"
```
Comment (robinkeunen) [inline cooperator/i18n/cooperator.pot:84]: This may work but I see no occurrences of this in Odoo codebase

Comment (robinkeunen) [reply cooperator/i18n/cooperator.pot:84]: => in a lot of other lower instances

File: cooperator/i18n/cooperator.pot [hunk 4]
```diff
@@ -384,26 +393,28 @@ msgstr ""
...
-"                Capital (Ref ${object.number or 'n/a'})\n"
+msgid ""
+"${object.company_id.name} Request to Release\n"
+"                Capital (Ref ${object.name or 'n/a'})\n"
 "            "
 msgstr ""
 
 #. module: cooperator
-#: code:addons/cooperator/models/subscription_request.py:69
+#: code:addons/cooperator/models/subscription_request.py:0
```
Comment (robinkeunen) [inline cooperator/i18n/cooperator.pot:403]: Still at line 69 in odoo 12.

File: cooperator/i18n/cooperator.pot [hunk 5]
```diff
@@ -384,26 +393,28 @@ msgstr ""
...
 #, python-format
 msgid "%s is not available to companies"
 msgstr ""
 
 #. module: cooperator
-#: code:addons/cooperator/models/subscription_request.py:74
+#: code:addons/cooperator/models/subscription_request.py:0
```
Comment (robinkeunen) [inline cooperator/i18n/cooperator.pot:409]: same here.

File: cooperator/i18n/cooperator.pot [hunk 6]
```diff
@@ -384,26 +393,28 @@ msgstr ""
...
 #, python-format
 msgid "%s is not available to individuals"
 msgstr ""
 
 #. module: cooperator
 #: model_terms:ir.ui.view,arch_db:cooperator.theme_invoice_G002_document
-msgid "<span groups=\"account.group_show_line_subtotals_tax_excluded\">\n"
+msgid ""
+"<span groups=\"account.group_show_line_subtotals_tax_excluded\">\n"
```
Comment (robinkeunen) [inline cooperator/i18n/cooperator.pot:417]: This looks strange.

File: cooperator/i18n/cooperator.pot [hunk 7]
```diff
@@ -515,24 +528,18 @@ msgid "<strong>Your Contact:</strong>"
 msgstr ""
 
 #. module: cooperator
-#: code:addons/cooperator/wizard/create_subscription_from_partner.py:205
+#: code:addons/cooperator/wizard/create_subscription_from_partner.py:0
```
Comment (robinkeunen) [inline cooperator/i18n/cooperator.pot:531]: same here

File: cooperator/i18n/cooperator.pot [hunk 8]
```diff
@@ -945,7 +940,7 @@ msgid "Cooperator Candidates"
 msgstr ""
 
 #. module: cooperator
-#: model:ir.actions.act_window,name:cooperator.account_invoice_action
+#: model:ir.actions.act_window,name:cooperator.action_move_out_invoice_type
```
Comment (robinkeunen) [inline cooperator/i18n/cooperator.pot:943]: Still account_invoice_action

File: cooperator/i18n/cooperator.pot [hunk 9]
```diff
@@ -1026,14 +1014,15 @@ msgid "Create Subscription From Partner"
 msgstr ""
 
 #. module: cooperator
-#: model_terms:ir.actions.act_window,help:cooperator.account_invoice_action
+#: model_terms:ir.actions.act_window,help:cooperator.action_move_out_invoice_type
```
Comment (robinkeunen) [inline cooperator/i18n/cooperator.pot:1017]: idem

File: cooperator/i18n/cooperator.pot [hunk 10]
```diff
@@ -1235,12 +1223,7 @@ msgid "Entry type"
 msgstr ""
 
 #. module: cooperator
-#: model:account.account,name:cooperator.account_equity_demo
-msgid "Equity"
-msgstr ""
-
-#. module: cooperator
-#: code:addons/cooperator/wizard/update_share_line.py:45
+#: code:addons/cooperator/wizard/update_share_line.py:0
```
Comment (robinkeunen) [inline cooperator/i18n/cooperator.pot:1226]: idem

File: cooperator/i18n/cooperator.pot [hunk 11]
```diff
@@ -1295,20 +1283,14 @@ msgstr ""
 #. module: cooperator
 #: model:ir.model.fields,field_description:cooperator.field_product_product__force_min_qty
 #: model:ir.model.fields,field_description:cooperator.field_product_template__force_min_qty
-msgid "Force Min Qty"
+msgid "Force minimum quantity?"
```
Comment (robinkeunen) [inline cooperator/i18n/cooperator.pot:1286]: Need a space before ?

File: cooperator/i18n/cooperator.pot [hunk 12]
```diff
@@ -1450,8 +1434,8 @@ msgid "Is an operation"
 msgstr ""
 
 #. module: cooperator
-#: model:ir.model.fields,field_description:cooperator.field_partner_update_info__is_company
-msgid "Is company"
+#: model:ir.model.fields,field_description:cooperator.field_partner_create_subscription__is_company
+msgid "Is company?"
```
Comment (robinkeunen) [inline cooperator/i18n/cooperator.pot:1438]: idem

File: cooperator/i18n/fr.po [hunk 13]
```diff
@@ -4,24 +4,24 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 12.0\n"
+"Project-Id-Version: Odoo Server 14.0\n"
```
Comment (robinkeunen) [inline cooperator/i18n/fr.po:7]: idem

File: cooperator/i18n/fr.po [hunk 14]
```diff
@@ -4,24 +4,24 @@
...
 "Plural-Forms: \n"
-"X-Generator: Poedit 3.0.1\n"
+"X-Generator: Poedit 3.1.1\n"
 
 #. module: cooperator
 #: model:mail.template,report_name:cooperator.email_template_release_capital
 msgid ""
 "\n"
-"                ${(object.number or '').replace('/','_')}_${object.state == 'draft' and 'draft' or ''}\n"
+"                ${(object.name or '').replace('/','_')}_${object.state == 'draft' and 'draft' or ''}\n"
```
Comment (robinkeunen) [inline cooperator/i18n/fr.po:24]: idem

File: cooperator/i18n/fr.po [hunk 15]
```diff
@@ -117,7 +114,93 @@ msgid ""
...
+"    <p>We confirm the reception of you payment for the new share(s) you have taken.</p>\n"
+"\n"
+"    <br/>\n"
+"    <p>Find in attachment your ${object.env.company.name} certificate.</p>\n"
+"    <p>Thank you for trusting ${object.env.company.name or 'us'}!</p>\n"
+"    <br/>\n"
+"    <p>Sustainably your,</p>\n"
+"    <p>${object.env.company.name}.</p>\n"
+"\n"
+"    % if object.env.company.street:\n"
```
Comment (robinkeunen) [inline cooperator/i18n/fr.po:135]: I'd be careful with these `object.env.company` as well.

File: cooperator/i18n/fr.po [hunk 16]
```diff
@@ -255,19 +552,16 @@ msgid ""
 "    </div>\n"
 "</div>\n"
 "            "
-msgstr ""
 
 #. module: cooperator
-#: model:mail.template,body_html:cooperator.email_template_confirmation_company
+#: model:mail.template,body_html:cooperator.email_template_confirmation
```
Comment (robinkeunen) [inline cooperator/i18n/fr.po:557]: Probably still `email_template_confirmation_company`

File: cooperator/i18n/fr.po [hunk 17]
```diff
@@ -394,25 +683,62 @@ msgid ""
...
+"    </div>\n"
+"</div>\n"
+"            "
 
 #. module: cooperator
 #: model:mail.template,subject:cooperator.email_template_release_capital
 msgid ""
 "${object.company_id.name} Request to Release\n"
-"                Capital (Ref ${object.number or 'n/a'})\n"
+"                Capital (Ref ${object.name or 'n/a'})\n"
```
Comment (robinkeunen) [inline cooperator/i18n/fr.po:730]: idem

File: cooperator/i18n/fr.po [hunk 18]
```diff
@@ -959,7 +1269,7 @@ msgid "Cooperator Candidates"
 msgstr "Candidats coopérateur"
 
 #. module: cooperator
-#: model:ir.actions.act_window,name:cooperator.account_invoice_action
+#: model:ir.actions.act_window,name:cooperator.action_move_out_invoice_type
```
Comment (robinkeunen) [inline cooperator/i18n/fr.po:1272]: idem.

File: cooperator/i18n/fr.po [hunk 19]
```diff
@@ -1034,15 +1343,13 @@ msgid "Create Subscription From Partner"
 msgstr "Créer une demande de souscription depuis le contact"
 
 #. module: cooperator
-#: model_terms:ir.actions.act_window,help:cooperator.account_invoice_action
+#: model_terms:ir.actions.act_window,help:cooperator.action_move_out_invoice_type
```
Comment (robinkeunen) [inline cooperator/i18n/fr.po:1346]: idem

File: cooperator/i18n/fr.po [hunk 20]
```diff
@@ -1906,7 +2222,7 @@ msgid "Register Number Operation"
 msgstr "Numéro d'opération"
 
 #. module: cooperator
-#: model:ir.ui.menu,name:cooperator.menu_account_invoice_action
+#: model:ir.ui.menu,name:cooperator.menu_move_out_invoice_type
```
Comment (robinkeunen) [inline cooperator/i18n/fr.po:2225]: idem

**Review by robinkeunen — CHANGES_REQUESTED** [review]
I think it's a bit dangerous to just copy-paste translations from versions to versions.

**Comment by robinkeunen** [comment]
Discussed with @victor-champonnois : the safer way to do this would be to start from scratch `.pot` and `fr.po` files. Since all current customers have translations in place and new one will be on 14.0, we won't spend time on this (at least not without weblate).

**Comment by huguesdk** [comment]
follow-up: #372.

**Pull request closed without merging**